src/validations/html/htmlInspector.js

```diff
@@ -2,14 +2,36 @@ import last from 'lodash/last';
 import isNull from 'lodash/isNull';
 import Validator from '../Validator';
 
+const specialCases = {
+  li: {
+    reason: 'invalid-tag-parent',
+    parent: '<ul>, <ol>, or <menu>',
```

Comment:
Couldn't figure out an easy way to do this, so I changed it to the more robotic sounding "The <li> tag needs to be inside one of the following tags: <ul>, <ol>, <menu>."